istioctl/cmd/describe.go

```diff
@@ -623,21 +625,40 @@ func printAuthn(writer io.Writer, pod *v1.Pod, debug envoy_v2.AuthenticationDebu
 		return
 	}
 
-	mTLSType := map[string]string{
+	mTLSType13 := map[string]string{
 		"HTTP":        "HTTP",
 		"mTLS":        "STRICT",
 		"HTTP/mTLS":   "PERMISSIVE",
 		"TLS":         "SIMPLE",
 		"custom mTLS": "custom mTLS",
 		"UNKNOWN":     "Unknown",
 	}
-	tlsType, ok := mTLSType[debug.ServerProtocol]
+	tlsType, ok := mTLSType13[debug.ServerProtocol]
 	if !ok {
 		tlsType = debug.ServerProtocol
 	}
 
```

Comment:
Move these lines to line 646 after the conflictStatus == "AUTO" check.

Reply:
Fixed